README.md

```diff
@@ -0,0 +1,74 @@
+# ostext-style-guide
+
+Design Guide is an application used by OpenStax as a living style guide document for our book templates.
+
+## Requirements
+You will need the following properly installed on your computer.
+
+* [Git](http://git-scm.com/)
+* [Node.js](http://nodejs.org/) (with NPM)
+* [Python 2.7](https://www.python.org/downloads/) or above
+  * If using version 2.7 then you must install [subprocess32](https://pypi.python.org/pypi/subprocess32)
```

Comment:
Can `subprocess32` be installed in the virtualenv so it does not have to be installed globally?